paddle/fluid/framework/executor.cc

```diff
@@ -58,10 +59,27 @@ ExecutorPrepareContext::ExecutorPrepareContext(
 
 void ExecutorPrepareContext::PrepareUnusedVars(
     const std::vector<std::string>& keep_vars, bool force_disable_gc) {
+#ifdef PADDLE_WITH_NGRAPH
+  if (FLAGS_use_ngraph) {
+    LOG_FIRST_N(WARNING, 1)
+        << "FLAGS_use_ngraph=True, garbage collection strategy is "
+           "disabled in Executor";
+    force_disable_gc = true;
+  }
```

Comment:
I do not know why, but when `FLAGS_use_ngraph=True` and gc is enabled, there is difference in ngraph op unittests. Maybe there is some cache inside ngraph engine?